### PR TITLE
Awesome books: with ES6

### PR DESCRIPTION
**Pull request by brighton0102**
### In this milestone I: 

- Use ES6 modules to write modular JavaScript.
- Create a new directory and initialize it with git.
- Use the npm init -y command to create a package.json file.
- Create the entry point for your JavaScript code called index.js
- Create an index.css file for your styles.
- Create the main index.html file. Link your index.js and index.css files there.
- Create a directory called modules.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      e.preventDefault();
+      bookManager.addNewData();
+    });
+  }
+}
+
+const app = new App();
+export default app;
+
+// export {DisplaySections, bookManager, initVisitedMenu}
```
Comment (DammyShittu) [inline index.js]: - [ ] Kindly remove the commented line of code as it is bad practice to commit commented code. You can visit git history to view old code.

File: modules/dateModule.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    this.updateTime();
+    setInterval(() => {
+      this.updateTime();
+    }, 1000);
+  }
+
+  updateTime = () => {
+    const dt = DateTime.utc().toLocal();
+    this.dateTimeElement.innerHTML = dt.toLocaleString(DateTime.DATETIME_MED_WITH_SECONDS);
+  }
```
Comment (DammyShittu) [inline modules/dateModule.js:19]: - **[OPTIONAL]** Arrow functions are not often advised in classes. They have binding issues and are usually not part of the prototype. It is best to retain the `function` keyword methods in classes. You can change them back in the other js files where you have arrow functions in classes

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @brighton0102,

## Highlights
✔️ Arrow functions

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joel-kalema — APPROVED** [review]
Hi @brighton0102 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 


## Highlights

The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**